packages/elements/src/photon-multirx-form/photon-prescribe-workflow.tsx

```diff
@@ -74,6 +76,26 @@ export type PrescribeProps = {
   externalOrderId?: string;
 };
 
+export const ScreenDraftedPrescriptionsQuery = gql`
```

Comment:
imo, I prefer to keep this in the `components` package, probably the responsibility of the `ScreeningAlerts` component, keep it close to the component that needs it rather than passing the info through

Reply:
will refactor

Reply:
would you be OK if this refactor was done early next week in a separate PR?

Reply:
yes totally